src/Adapter/Category/CommandHandler/BulkDeleteCategoriesHandler.php

```diff
@@ -62,7 +63,13 @@ public function handle(BulkDeleteCategoriesCommand $command)
                 throw new FailedToDeleteCategoryException(sprintf('Failed to delete category with id %s', var_export($categoryId->getValue(), true)));
             }
 
-            $this->handleProductsUpdate((int) $category->id_parent, $command->getDeleteMode());
+            $deletedCategoryIdsByParent[(int) $category->id_parent] = [$categoryId->getValue()];
```

Comment:
Careful here because you don't if the parent you are storing here may be one of the deleted categories (or one of their children) You need to make sure that all these parent IDs still match existing categories